target/postfix/master.cf

```diff
@@ -73,3 +73,4 @@ virtual        unix  -       n       n       -       -       virtual
 lmtp           unix  -       -       n       -       -       lmtp
 anvil          unix  -       -       n       -       1       anvil
 scache         unix  -       -       n       -       1       scache
+postlog  unix-dgram  n       -       n       -       1       postlogd
```

Comment:
This should be fine to keep, aligning with upstream `master.cf`. [`postlogd`](https://www.postfix.org/postlogd.8.html) explains that it's mostly for when you don't want to use syslog, by configuring `mail.cf:maillog_file`. [`postlog`](https://www.postfix.org/postlog.1.html) handles the logging, and defaults to syslog.